test/service/test_manifest.py

```diff
@@ -350,6 +350,13 @@ def test_manifest_zarr(self):
                     self.assertEqual(11, len({row['file_sha256'] for row in rows}), 'Incorrect unique sha256 count')
                     self.assertEqual(11, len({row['file_crc32c'] for row in rows}), 'Incorrect unique crc32c count')
 
+                    response = self._get_manifest(ManifestFormat.curl, filters)
+                    self.assertEqual(200, response.status_code, 'Unable to download manifest')
+
+                    lines = response.content.decode('utf-8').splitlines()
+                    output_paths = {line.partition('=')[-1] for line in lines if line.startswith('output')}
```

Comment:
I found the `[-1]` tuple index here confusing. I think usually it's better to explicitly unpack a tuple. Indexing here is convenient for the comprehension, but I'm not convinced that it is necessary since you're not actually using the output_paths. You could remove the index and just assert the line count?

Reply:
Done, I missed that! :)